addons/l10n_co/data/template/account.tax-co.csv

```diff
@@ -1,225 +1,225 @@
-"","","","","","","","","","tax","invoice","","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","","","",""
-"l10n_co_tax_53","1","2.5% Rte","-2.5","percent","sale","tax_group_r_ren_25","","","base","invoice","","RteFte -2.5% Sales","-2.5%","RteFte -2.50% Ventas"
-"","","","","","","","","","tax","invoice","co_puc_135515","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","co_puc_135515","","",""
-"l10n_co_tax_54","1","3.50% Rte","-3.5","percent","sale","tax_group_r_ren_35","","","base","invoice","","RteFte -3.50% Sales","-3.5%","RteFte -3.50% Ventas"
-"","","","","","","","","","tax","invoice","co_puc_135515","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","co_puc_135515","","",""
-"l10n_co_tax_55","1","0.75% Rte","-0.75","percent","sale","tax_group_r_iva_075","","","base","invoice","","RteIVA 15% over 5% VAT Sales","-0.75%","RteIVA 15% sobre el 5% IVA Ventas"
-"","","","","","","","","","tax","invoice","co_puc_135517","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","co_puc_135517","","",""
-"l10n_co_tax_56","1","2.85% Rte","-2.85","percent","sale","tax_group_r_iva_285","","","base","invoice","","RteIVA 15% over 19% VAT Sales","-2.85%","RteIVA 15% sobre el 19% IVA Ventas"
-"","","","","","","","","","tax","invoice","co_puc_135517","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","co_puc_135517","","",""
-"l10n_co_tax_57","1","0.414% Rte","-0.414","percent","sale","tax_group_r_ica_0414","","","base","invoice","","RteICA -0.414% Sales","-0.414%","RteICA -0.414% Ventas"
-"","","","","","","","","","tax","invoice","co_puc_135518","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","co_puc_135518","","",""
-"l10n_co_tax_58","1","0.966% Rte","-0.966","percent","sale","tax_group_r_ica_0966","","","base","invoice","","RteICA -0.966% Sales","-0.966%","RteICA -0.966% Ventas"
-"","","","","","","","","","tax","invoice","co_puc_135518","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","co_puc_135518","","",""
-"l10n_co_tax_covered_goods","1","0% C A","0.0","percent","sale","tax_group_covered_goods","","","base","invoice","","Covered Assets","0%","Bienes Cubiertos"
-"","","","","","","","","","tax","invoice","","","",""
-"","","","","","","","","","base","refund","","","",""
-"","","","","","","","","","tax","refund","","","",""
+"id","sequence","name","amount","amount_type","type_tax_use","tax_group_id","active","children_tax_ids","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","description","invoice_label",name@es,description@es
```

Comment:
```suggestion
"id","sequence","name","amount","amount_type","type_tax_use","tax_group_id","active","children_tax_ids","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/account_id","description","invoice_label","name@es","description@es"
```

Reply:
this is interesting actually because adding double quotes around the name breaks the CoA